docs/Networking.asciidoc

```diff
@@ -238,6 +238,10 @@ Also, the `firewall-config` GUI tool for firewalld can be used for configuration
 WORKER_CLASS = qemu_x86_64,tap
 ---------------
 
+NOTE: The number of tap devices should correspond to the number of the
+running workers. For example, if you have set up 3 tab devices, the analogous
```

Comment:
Not sure whether the use of 'analogous' is appropriate here. I'd just use 'same number of'.